nat-lab/tests/utils/linux_router.py

```diff
@@ -138,31 +162,13 @@ async def delete_vpn_route(self):
                 raise exception
 
     async def create_exit_node_route(self) -> None:
-        await self._connection.create_process(
-            [
-                "iptables",
-                "-t",
-                "nat",
-                "-A",
-                "POSTROUTING",
-                "-s",
-                "100.64.0.0/10",
-                "!",
-                "-o",
-                self._interface_name,
-                "-j",
-                "MASQUERADE",
-            ],
-        ).execute()
-
-    async def delete_exit_node_route(self) -> None:
-        try:
+        if self.ip_stack in [IPStack.IPv4, IPStack.IPv4v6]:
             await self._connection.create_process(
                 [
                     "iptables",
                     "-t",
                     "nat",
-                    "-D",
+                    "-A",
```

Comment:
Why is this changed from D to A here ?

Reply:
The `git` changes are just projected weirdly here. If you'll check the functions, `create_exit_node_route` has `-A`, and `delete_exit_node_route` has `-D` options.